solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebookEnvironmentLifecycleService.ts

```diff
@@ -32,8 +32,13 @@ export default class SagemakerNotebookEnvironmentLifecycleService implements Env
 
     const { datasetsBucketArn, mainAccountRegion, mainAccountId, mainAcctEncryptionArn } =
       await this.helper.getCfnOutputs();
+
+    const datasetIds: string[] =
+      envMetadata.DATASETS?.map((dataSet: { id: string }) => {
```

Comment:
Did you add a test for multiple datasets as well?

Reply:
I think we only need to worry about the case where its empty (which is tested in `Launch should return a mocked id` and where its non-empty. If we've got a reason to test specific sizes though just let me know and I'm happy to add it.